ibis/backends/dask/executor.py

```diff
@@ -364,10 +364,17 @@ def visit(cls, op: ops.Sort, parent, keys):
         # 2. sort the dataframe using those columns
         # 3. drop the sort key columns
         ascending = [key.ascending for key in op.keys]
+        na_pos_dict = {True: "first", False: "last"}
+        na_position = na_pos_dict[all(key.nulls_first for key in op.keys)]
```

Comment:
pandas doesn't support a list that contains different options for null orders. It only accepts `na_position`= "last" or "first".

What do we  want to do here? we could check that if there is more than one to break.